### ISSUE TITLE
Timeout while unintended evaluation of integral without any concrete variables

### ISSUE THREAD

**Issue by Famondir** [maths/moodle-qtype_stack#982]
Hi all,
I have a STACK question where the students should input the following formla `sqrt(1/T*int(u(t)^2,t,0,T))`. But I can't set this as teachers answer because then I get an Time out when I try to save it, when all variables like `T, u(t)` are not defined. It seems he tries to evaluate it anyway. I tried to surpress it with turning simplification off but that wasn't the solution.

When I set one (or both) of the variables like `T=1, u(t):= 1;` or `u(t):= t;` it works but I just want to check the formula not any concrete result. Is there a way to prohib the evaluation of the teacher answer `TAns1: sqrt(1/T*int(u(t)^2,t,0,T));`?

Maybe something like: `ev(TAns1: sqrt(1/T*int(u(t)^2,t,0,T)), noeval = true)`

I already tried the apostrophe from `'diff()` in `'int()` but that did not work.

Kind regards
Simon

**Comment by anst-i** [comment]
I've tried entering `sqrt(1/T*int(u(t)^2,t,0,T));` into the STACK sandbox: Maxima asks `Is T positive, negative or zero?`, and since STACK doesn't answer, there's a timeout. It worked once I wrote `assume_pos: true;` or `assume( T > 0);` in the line just before, at least in the sandbox. You might want to try one of these.

Best, Andreas

PS: For STACK usage questions, you might want to join our [Community chat](https://stack-assessment.zulipchat.com/) instead of the development issue tracker.
